server/templates/place.html

```diff
@@ -17,7 +17,15 @@
 
 {% set main_id = 'dc-places' %}
 {% set page_id = 'page-dc-places' %}
-{% set title = place_name + ' ' + category + ' - ' + _('Place Explorer') %}
+{% set category_title = place_name + ' ' + category %}
```

Comment:
did you mean to drop `_('Place Explorer') ` ?

Reply:
Originally intentional to match Adriana's suggestions. Reverted to original title (see discussion below)